### ISSUE TITLE
Rpmverifyfile probe can't verify files from /bin directory

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#170]
Hello, I was writing a small test for rpmverifyfile probe. I tried to find a filepath that exists on every system so it is suitable for an upstream test. I suggested `/usr/bin/sh`, which worked well, but @mpreisler thought that `/bin/sh` would be better, because not all the distros do the usrmove.

Unfortunately, setting `/bin/sh` as filepath in **//lin-def:rpmverifyfile_object/lin-def:filepath** does not work! I got that _object does not exist_ in OVAL results. The same problem I have for `/bin/bash`, `/bin/python`, `/bin/diff`. I think it is very likely that this issue affects whole `/bin` directory.

The problem is that this behavior is not consistent with running `rpm -Vf` command manually. For example:

``` bash
[jcerny@thinkpad ~]$ rpm -Vf /bin/bash
[jcerny@thinkpad ~]$ echo $?
0
```

Works OK. 

If the object does not exist in a rpm package, I got an error message that file is not owned by any package:

``` bash
[jcerny@thinkpad ~]$ rpm -Vf /home/jcerny/pokus.txt
soubor /home/jcerny/pokus.txt nevlastní žádný balíček
[jcerny@thinkpad ~]$ echo $?
1
```

This issue might be related to https://fedorahosted.org/openscap/ticket/512

My problem is with OpenSCAP 1.2.5 on Fedora 22 and also with current maint-1.2

**Comment by jan-cerny** [comment]
It doesn't seem to be related to https://fedorahosted.org/openscap/ticket/512, because that ticket was a bug in content, not in OpenSCAP.

**Comment by evgenyz** [comment]
@jan-cerny I guess this bug was related to symlinks and was fixed in 1.3.2. I'm I right?

**Comment by jan-cerny** [comment]
Great discovery. I have tried it and the issue isn't reproducible anymore. It has been fixed by 3beda86f52aa78ce1d7cc91a911390084bb66a28.
